package.json

```diff
@@ -33,6 +36,7 @@
     "openzeppelin-test-helpers": "^0.2.0",
     "prettier": "^1.16.4",
     "solhint": "^2.0.0",
-    "truffle": "^5.0.8"
+    "truffle": "^5.0.8",
+    "truffle-flattener": "webmaster128/truffle-flattener#whitespacing"
```

Comment:
Interesting link on the readme to https://github.com/nomiclabs/buidler

I don't think we need to switch build systems (truffle works for us, let's stick with it), but curious about the option.